### ISSUE TITLE
Python 3.9 - object Lock can't be used in 'await' expression

### ISSUE THREAD

**Issue by michaeldavie** [pyinsteon/pyinsteon#111]
This line appears to me incompatible with Python 3.9 and is causing the Home Assistant Insteon integration to fail with the error below.

https://github.com/pyinsteon/pyinsteon/blob/7025cb6f3beb42f43c6a13799eff0a6b77e38e69/pyinsteon/protocol/http_transport.py#L119

```
2021-07-11 11:31:44 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/pyinsteon/protocol/protocol.py", line 187, in _write_messages
    await self._transport.async_write(msg)
  File "/usr/local/lib/python3.9/site-packages/pyinsteon/protocol/http_transport.py", line 110, in async_write
    await self._async_write_url(url=url, msg=bytes(data))
  File "/usr/local/lib/python3.9/site-packages/pyinsteon/protocol/http_transport.py", line 119, in _async_write_url
    await self._read_write_lock
TypeError: object Lock can't be used in 'await' expression
```

**Comment by bshep** [comment]
Looks like the key is replacing:
`await self._read_write_lock`
with
`async with self._read_write_lock`

Will probably resilve the issue ( i have no way of testing since for some reason i cant get custom packages to install on my HA instance )

**Comment by rlust** [comment]
How can I make this change to get it working on my HASSIO system?

**Comment by rlust** [comment]
I just upgraded to 2021.7.2 and it did not fix the Insteon issue. I still cannot load the Hub 2245 integration. Still says device not found.

**Comment by rlust** [comment]
I just upgraded to 2021.7.2 and it did not fix the Insteon issue. I still cannot load the Hub 2245 integration. Still says device not found.

**Comment by bshep** [comment]
> I just upgraded to 2021.7.2 and it did not fix the Insteon issue. I still cannot load the Hub 2245 integration. Still says device not found.

The library has not been updated on 2021.7.2, theres an open request to update it but it hasnt been done yet.

**Comment by rlust** [comment]
2021.7.3 Fixed the Insteon hub V2 issues!!

Thanks for the support!!

**Comment by michaeldavie** [comment]
Fixed in #114
